src/mcts/search.cc

```diff
@@ -355,6 +355,46 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
                             : "");
     infos.emplace_back(oss.str());
   }
+
+  // Include stats about the node in similar format to its children above.
+  std::ostringstream oss;
+  oss << std::fixed << std::right << "node ";
+  oss << " (" << std::setw(4) << node->GetNumEdges() << ")";
+  oss << " N: " << std::setw(7) << node->GetN() << " (+" << std::setw(2)
+      << node->GetNInFlight() << ") ";
+  oss << "(P: " << std::setw(5) << std::setprecision(2)
+      << node->GetVisitedPolicy() * 100 << "%) ";
+  oss << "(WL: " << std::setw(8) << std::setprecision(5) << node->GetWL()
+      << ") ";
+  oss << "(D: " << std::setw(6) << std::setprecision(3) << node->GetD() << ") ";
+  oss << "(M: " << std::setw(4) << std::setprecision(1) << node->GetM() << ") ";
+  oss << "(Q: " << std::setw(8) << std::setprecision(5)
+      << node->GetQ(-draw_score) << ") ";
+  oss << "(V: ";
```

Comment:
It feels that this code would benefit from some de-duplication.
At least for (V) and flags.. But potentially for some other values too (not (S) and (U) and probably not (N) as it has to be the first).

Maybe have a helper function/lambda, something like:
```
auto print_tail[](std::ostream* os, const Node* node) {
   // Node may be nullptr.
   if (node) {
     oss << "(WL: " << std::setw(8) << std::setprecision(5) << node->GetWL() << ") ";
     oss << "(D: " << std::setw(6) << std::setprecision(3) << node->GetD() << ") ";
   } else {
     oss << "(WL:---) (D:---) ..."; // Not sure if needed at all.
   }
}
```

Reply:
Deduplicated with 3 lambdas except for move/index, S and U.

Reply:
I kept the Q = fpu for unvisited.